packages/lyra/src/lyra.ts

```diff
@@ -129,7 +135,7 @@ export class Lyra<TSchema extends PropertiesSchema = PropertiesSchema> {
               break;
             }
 
-            const fullDoc = this.docs.get(id);
+            const fullDoc = this.docs.get(id)!;
```

Comment:
@micheleriva Is there a better way to do that? I hate asserting non-null

Reply:
mmm afaik this is the only way... if there's a guarantee that the value exists, it's not that bad